caf-logging-logback/src/test/java/com/github/cafapi/logging/logback/LogInjectionTests.java

```diff
@@ -113,8 +113,8 @@ public void testLogExceptionMessage() throws Exception
 
         public StdOutRedirector(final OutputStream out)
         {
-            this.originalStdOut = System.out;
-            System.setOut(new PrintStream(out));
+            this.originalStdOut = System.err;
```

Comment:
The variable is called `originalStdOut` and it is in a class called `StdOutRedirector`.